variables.tf

```diff
@@ -35,3 +35,35 @@ variable "region" {
     error_message = "Invalid input, options: \"us-south\", \"ca-tor\", \"eu-es\", \"eu-de\"."
   }
 }
+
+variable "cos_instance_crn" {
+  type        = string
+  description = "CRN of the Cloud Object Storage to store SCC data"
+
+  validation {
+    condition     = var.cos_instance_crn != null
+    error_message = "Please provide COS instance CRN to store SCC data"
+  }
+}
+
+variable "cos_bucket" {
+  type        = string
+  description = "The name of the Cloud Object Storage bucket to be used in SCC instance"
+
+  validation {
+    condition     = var.cos_bucket != null
+    error_message = "Please provide COS bucket to store SCC data"
+  }
+}
+
+variable "en_instance_crn" {
+  type        = string
+  default     = ""
+  description = "The CRN of Event Notifications instance to be used with SCC"
```

Comment:
Also mention is no value is passed, Event Notifications will not be enabled for the instance